thread/source/lib.c

```diff
@@ -10,7 +10,7 @@ void *memset(void * ptr, int value, unsigned int num){
 }
 
 int strcomp(char* s1, char* s2, unsigned int size){
-	for(int i = 0; i < size; i++){
+	for(int i = 0; i < size*2; i++){	// doubling size brings this in line with strcopy() and fixes the "Rei" string randomly replacing "Ver" in other places.
```

Comment:
Short of rewriting the way this is done completely, the other alternative is edit line 71 of thread.c from

		if(strcomp((void*)0x27B00000  - i, (void*)L"Ver.", 4)) strcopy((void*)0x27B00000 - i, (void*)L"\uE024Rei", 4);

to

		if(strcomp((void*)0x27B00000  - i, (void*)L"Ver.", 8)) strcopy((void*)0x27B00000 - i, (void*)L"\uE024Rei", 4);

This would effectively result in the same fix, but my understanding was that addressing the strcomp() function directly was a better way to fix the issue.